x-pack/plugin/core/src/main/resources/metrics-mappings.json

```diff
@@ -2,6 +2,25 @@
   "template": {
     "mappings": {
       "dynamic_templates": [
+        {
+          "match_ip": {
+            "match_mapping_type": "string",
+            "match": "ip",
+            "mapping": {
+              "type": "ip"
+            }
+          }
+        },
+        {
+          "match_message": {
+            "match_mapping_type": "string",
+            "match": "message",
+            "mapping": {
+              "type": "text",
+              "norms": false
+            }
+          }
+        },
```

Comment:
Do we really messages for metrics?

Reply:
@exekias Do you know how often we use it?

One of the main reasons I kept it in is to make sure it is compatible with ECS and not accidentally becomes a keywordl

Reply:
I asked @exekias in a different channel, who told me that metrics datasets cannot have a message field.